rskj-core/src/main/java/co/rsk/rpc/netty/RskWebSocketJsonRpcHandler.java

```diff
@@ -84,7 +107,13 @@ protected void channelRead0(ChannelHandlerContext ctx, ByteBufHolder msg) {
         // delegate to the next handler if the message can't be matched to a known JSON-RPC request
         ctx.fireChannelRead(msg);
     }
-
+    
+    private String getJsonWithTypedId(JsonNode jsonNodeRequest, JsonRpcIdentifiableMessage response) throws JsonProcessingException {
```

Comment:
Maybe a clarifying comment here would be nice. 💪🏼

Reply:
added